packages/e2e-tests/specs/embedding.test.js

```diff
@@ -229,6 +231,21 @@ describe( 'Embedding content', () => {
 		await page.waitForSelector( 'figure.wp-block-embed-twitter' );
 	} );
 
+	it( 'embed blocks should work inside blocks with locking all', async () => {
+		await activatePlugin( 'gutenberg-test-innerblocks-locking-all-embed' );
+		await createNewPost();
+		await insertBlock( 'Test Inner Blocks Locking All Embed' );
+		const embedInputSelector = '.components-placeholder__input[aria-label="Embed URL"]';
+		await page.waitForSelector( embedInputSelector );
+		await page.click( embedInputSelector );
+		// This URL can't be embedded, but without the trailing slash, it can.
```

Comment:
Should URL have the trailing slash? This comment is confusing.

Reply:
We need the trailing slash the comment was clarified 👍

Reply:
Looking at code I assume, we don't need it
Travis is happy 🤷‍♂